source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -97,19 +97,26 @@ class CircuitBreakerSerializer(serpy.DictSerializer):
     reset_timeout = Field(schema_type=int, display_none=True)
 
 
-class StreetNetworkSerializer(serpy.DictSerializer):
+class OutsideServiceCommon(serpy.DictSerializer):
     class_ = Field(schema_type=str, label='class', attr='class')
-    id = Field(schema_type=str)
+    _id = Field(attr='id', schema_type=str)
+    circuit_breaker = MethodField(schema_type=CircuitBreakerSerializer, display_none=False)
+
+    def get_circuit_breaker(self, obj):
+        o = obj.get('circuit_breaker', None)
+        return CircuitBreakerSerializer(o, display_none=False).data if o else None
+
+
+class StreetNetworkSerializer(OutsideServiceCommon):
     modes = StringListField(display_none=True)
     timeout = MethodField(schema_type=float, display_none=False)
-    circuit_breaker = MethodField(schema_type=CircuitBreakerSerializer, display_none=False)
 
     def get_timeout(self, obj):
         return obj.get('timeout', None)
 
-    def get_circuit_breaker(self, obj):
-        o = obj.get('circuit_breaker', None)
-        return CircuitBreakerSerializer(o, display_none=False).data if o else None
+
+class RidesharingServicesSerializer(OutsideServiceCommon):
+    service_url = Field(schema_type=str, display_none=False)
```

Comment:
it's better to not expose that kind of things, sometime we have to deal with services that doesn't any kind of authentication, so we can't publicly show their url.

Reply:
ok

Reply:
good point